src/hb-aat-layout-lcar-table.hh

```diff
@@ -55,8 +55,7 @@ struct lcar
     if (caret_count)
     {
       hb_array_t<const HBINT16> arr = array.sub_array (start_offset, caret_count);
-      unsigned int count = arr.length;
-      for (unsigned int i = 0; i < count; ++i)
+      for (unsigned int i = 0; i < arr.length; ++i)
```

Comment:
Something inside stack allocated hb_array_t doesn't differ much with an explicitly stack allocated variable from optimizing standpoint I guess.

Reply:
Right.  But it's a pattern we have all over codebase.  Fine either way.  Just don't like mixing with other changes.

Reply:
Done